PSRT/Public/Find-RTTicket.ps1

```diff
@@ -0,0 +1,83 @@
+Function Find-RTTicket {
+    <#
+    .SYNOPSIS
+        Find RT tickets with a specified query
+    .DESCRIPTION
+        Find RT tickets with a specified query
+    .PARAMETER Query
+        Query string to pass to RT
+
+        Details: https://rt-wiki.bestpractical.com/wiki/REST#Ticket_Search
+    .PARAMETER Expand
+        If specified, pull ticket information for each ticket returned by this query
+    .PARAMETER Referer
+        Referer to use for whitelisting purposes.  See @ReferrerWhitelist RT configuration
+        
+        Defaults to 'https://<hostname>', which translates to <hostname>:443 on RT's side
+    .PARAMETER Session
+        RT session to use.  Defaults to PSRTConfig.Session (Created by New-RTSession)
+    .PARAMETER BaseUri
+        Base URI for RT.  Defaults to PSRTConfig.BaseUri
+    .PARAMETER Raw
+        If specified, do not parse output
+    .EXAMPLE
+        Find-RTTicket -Query "Created > '3 days ago'"
+    .EXAMPLE
+        Find-RTTicket -Query "Owner='wframe' AND Status='new'" -Referer http://WhiteListedHostname
+    .EXAMPLE
+        Find-RTTicket -Query "Status='new'" -Expand
+    .FUNCTIONALITY
+        Request Tracker
+    #>
+    [cmdletbinding()]
+    Param(
+        [parameter(Position = 1)]
+        [string]$Query,
+
+        [string]$Referer = "https://$ENV:COMPUTERNAME",
+
+        [switch]$Expand,
+
+        [Parameter( ValueFromPipeLine = $true, 
+            ValueFromPipelineByPropertyName = $true )] 
+        [ValidateNotNull()] 
+        [Microsoft.PowerShell.Commands.WebRequestSession] 
+        $Session = $PSRTConfig.Session,
+        [ValidateNotNull()] 
+        [string]$BaseUri = $PSRTConfig.BaseUri,
+        [switch]$Raw
+    )
+    $InvokeParams = @{ WebSession = $Session }
+    if($Referer)
+    {
+        $headers = @{}
+        $headers.Add('Referer', $Referer)
+        $InvokeParams.Add('Headers', $headers)
+    }
+    $Query = [System.Web.HttpUtility]::UrlEncode($Query)
```

Comment:
This handles encoding the URL so that odd characters can be used (we don't encode the whole URL, just the query part)